data/org.x.sticky.service.in

```diff
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.x.sticky
-Exec=/usr/bin/sticky
+Exec=@bindir@/sticky
```

Comment:
Would it be picked up by PATH on NixOS? Maybe no path/configure here at all, just 'sticky'?

Reply:
No, I got `org.freedesktop.DBus.Error.Spawn.ExecFailed` with that, I was suggested by [some docs](https://develop.kde.org/docs/features/d-bus/dbus_autostart_services/#creating-a-service-file) that d-bus services files have to use full path in `Exec` line :thinking: